### PR TITLE
SASS-5666 - 2. Create 'do you have more than 1 FHL?'

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -293,3 +293,10 @@ GET        /:taxYear/enhanced-structure-building-allowance/claim-amount
 POST       /:taxYear/enhanced-structure-building-allowance/claim-amount             controllers.EsbaClaimAmountController.onSubmit(taxYear: Int, mode: Mode = NormalMode)
 GET        /:taxYear/enhanced-structure-building-allowance/change-claim-amount      controllers.EsbaClaimAmountController.onPageLoad(taxYear: Int, mode: Mode = CheckMode)
 POST       /:taxYear/enhanced-structure-building-allowance/change-claim-amount      controllers.EsbaClaimAmountController.onSubmit(taxYear: Int, mode: Mode = CheckMode)
+
+GET        /:taxYear/fhl-intro                                                      controllers.furnishedholidaylettings.FhlIntroController.onPageLoad(taxYear: Int)
```
Comment (arunkumar461) [inline conf/app.routes:297]: Let's ensure uniformity in the URLs. So other pages can follow 
`/furnished-holiday-lettings/start `

File: conf/messages.en [hunk 2]
```diff
@@ -1132,3 +1132,21 @@ esbaClaimAmount.change.hidden = EsbaClaimAmount
 esbaClaimAmount.span = Enhanced structures and buildings allowance
 esbaClaimAmount.hint.individual = Enter the amount that you’ve calculated.
 esbaClaimAmount.hint.agent = Enter the amount that you’ve calculated for your client.
+
+fhlIntro.title = Furnished holiday lettings
+fhlIntro.heading = Furnished holiday lettings
+fhlIntro.summary = Understanding furnished holiday lettings
+fhlIntro.p = To qualify as a UK furnished holiday letting your property must be furnished and will also need to pass three occupancy conditions.
```
Comment (arunkumar461) [inline conf/messages.en:1139]: Agent content? your -> Client's Pls check the design once

File: app/forms/furnishedholidaylettings/FhlMoreThanOneFormProvider.scala [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import forms.mappings.Mappings
+import play.api.data.Form
+
+import javax.inject.Inject
+
+class FhlMoreThanOneFormProvider @Inject() extends Mappings {
+
+  def apply(individualOrAgent: String): Form[Boolean] =
+    Form(
+      "value" -> boolean(s"fhlMoreThanOne.error.required.$individualOrAgent")
```
Comment (arunkumar461) [inline app/forms/furnishedholidaylettings/FhlMoreThanOneFormProvider.scala:28]: Theres problem with naming as "value" -> check with Tosin

**Review by arunkumar461 — CHANGES_REQUESTED** [review]

File: conf/app.routes (hunk 1 above)
Comment (modivilioglu) [reply conf/app.routes:297]: done. thanks

File: app/forms/furnishedholidaylettings/FhlMoreThanOneFormProvider.scala (hunk 3 above)
Comment (modivilioglu) [reply app/forms/furnishedholidaylettings/FhlMoreThanOneFormProvider.scala:28]: yes, done.

File: conf/messages.en [hunk 4]
```diff
@@ -1129,3 +1129,39 @@ esbaClaimAmount.change.hidden = EsbaClaimAmount
...
+fhlIntro.heading = Furnished holiday lettings
+fhlIntro.summary = Understanding furnished holiday lettings
+
+fhlIntro.div.top.individual = You’ll need details of your expenses to answer the questions starting on the next page.
+fhlIntro.div.top.agent = You’ll need details of your client’s expenses to answer the questions starting on the next page.
+
+fhlIntro.p1.individual = To qualify as a UK furnished holiday letting your property must be furnished and will also need to pass three occupancy conditions.
+fhlIntro.p1.agent = To qualify as a furnished holiday letting your property must be furnished and based in the UK.
+
+fhlIntro.p2.individual =
```
Comment (arunkumar461) [inline conf/messages.en:1143]: No content for individual?

Comment (modivilioglu) [reply conf/messages.en:1143]: You're right. There's no content.

**Review by arunkumar461 — APPROVED** [review]

**Pull request merged**